cmd/runtimetest/main.go

```diff
@@ -291,7 +292,14 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
-func main() {
+func validate(context *cli.Context) {
```

Comment:
On Wed, Jun 08, 2016 at 03:42:11PM -0700, Mrunal Patel wrote:
> > -func main() {
> > +func validate(context *cli.Context) {
> 
> Needs to return error like #104 

Fixed in c2beb36 → 2c377ad.